### ISSUE TITLE
some question about parameter "offloading target"

### ISSUE THREAD

**Issue by uzmaru** [davidtw0320/Resources-Allocation-in-The-Edge-Computing-Environment#11]
Hi,david.Thank you for sharing code.But I have a question about parameter "offloading target". In Reinforcement learning, offloading target exists in both state and action.It's so confusing.May you explain it detailedly?Thanks.

**Comment by HuongDM1896** [comment]
Yes. I have same confuse too. And more. the offloading target is discrete variable, but DDPG only use for continuous action. Or you set the offloading target based on computing resource and bandwidth allocation? Example Bandwidth = 0 => offload = 0 (no offload). As I saw in source code, not this way to explain. I really confuse because, if this DDPG use for discrete, it's still a DDPG?

**Comment by HuongDM1896** [comment]
I have the answer to my question, It is softmax, you used the softmax to define the probability of offloading the target. It is a good solution to make the value of action continues.
